test/unit/inputs/input_test.rb

```diff
@@ -25,6 +25,27 @@
     end
   end
 
+  describe "marshalling" do
+    it "should be able to represent an Input as a Hash" do
+      input = Inspec::Input.new("test_input",
+        value: 12,
+        title: "Best input ever",
+        description: "important",
+        type: "Numeric",
+        required: true)
+      _(input.to_hash).must_equal({
+        name: "test_input",
+        options: {
```

Comment:
Alignment is a bit off here.

Reply:
I'm not sure what you're referring to - this was formatted by `chefstyle -a`, and passes `chefstyle` now. Could you maybe paste an example of what you mean?

Reply:
`options` starts a new hash, so the alignment needs to change.

```
_(input.to_hash).must_equal({
        name: "test_input",
        options: {
          value: 12,
          title: "Best input ever",
          description: "important",
          type: "Numeric",
          required: true,
          },
        })
```